molconfgen/output.py

```diff
@@ -0,0 +1,79 @@
+# Richard Richardson wrote this ;)
+# Function to add a box to the output of
+# sampler.generate_conformers
+
+import MDAnalysis as mda
+import MDAnalysis.transformations
+import numpy as np
+
+
+# Function for finding the largest radius in
+# the trajectory
+# this will come in handy later
+def largest_r(u):
+    r = 0
+    for ts in u.trajectory:
+        ag = u.atoms
+        rnew = ag.bsphere()[0]
+        if rnew > r:
+            r = rnew
+    return r
+
+def write_pbc_trajectory(u, filename, box = None):
+    ''' u is MDAnalysis universe with a trajectory
+     filename is the name of the trajectory output file
+     box argument describes how to set the box dimensions
+
+     I am not quite sure how to read a lone trajectory file
+     and sampler.generate_conformers returns a universe with
+     a trajectroy so for now I'm going to assume a universe 
+     with a trajectory '''
```

Comment:
Use `ag` as input. Then in the code write out the whole the universe. Make this clear in the docs.

Remove the text from the docs. That would be more of a comment.

Reply:
You can get the universe with
```python
u = ag.universe
```